gtsfm/configs/deep_front_end_colmap.yaml

```diff
@@ -0,0 +1,84 @@
+# Deep front-end configuration tuned for COLMAP datasets: https://colmap.github.io/datasets.html
```

Comment:
nit: might consider renaming this file `deep_front_end_colmapdata.yaml`. Currently it sounds a bit like we run colmap in this config. Could be nice also to explicitly mention "Gerrard Hall, Graham Hall, Person Hall, and South Building" in an inline comment, just to make this a little more self-contained.

Could we also add a sentence or two explaining what the main differences between this config and the current deep_front_end.yaml?

Reply:
Deleted this config